### ISSUE TITLE
NRF24L01+ connection unstable due to NRF24L01+ supply issue

### ISSUE THREAD

**Issue by ZufallsAnwendernameGeneratorGeraet** [tbnobody/OpenDTU#1105]
### What happened?

I have had issues with failed commands and failed updates in live view. Basically very unstable connection to the HM1500 .Then i thought i might add an small 10uF capacitor to the NRF24L01+ module. After that i had no issues any more. A quick search found that this is commonly suggested when using the NRF24L01 module. 

may be this should be mentioned in the README.md.

![Wiring_ESP32_Symbol](https://github.com/tbnobody/OpenDTU/assets/131549567/09c8cc71-1079-4ace-b30e-b65176721c7a)

### To Reproduce Bug

Connect an NRF24L01+ module by jumper wire to the ESP32 module

### Expected Behavior

Basically stable connection to the HM1500 (only a few meter distance).

### Install Method

Pre-Compiled binary from GitHub

### What git-hash/version of OpenDTU?

Any

**Comment by tbnobody** [comment]
The capacitor is mentioned in the troubleshooting area:

The NRF24L01+ needs relatively much current. With bad power supply (and especially bad cables!) a 10 µF capacitor soldered directly to the NRF24L01+ board connector brings more stability (pin 1+2 are the power supply). Note the polarity of the capacitor…

**Comment by his1220** [comment]
added the capactior and my problems with unknown commands and interruptions is gone. Thanks for the very great suggestion!!

**Comment by 112Feuerball112** [comment]
Could you upload a guide on how to expand the capacitor with charging?

**Comment by ZufallsAnwendernameGeneratorGeraet** [comment]
@112Feuerball112 
what do you exactly mean with "expand" the capacitor?

**Comment by tbnobody** [comment]
The capacitors are now also shown in the [documentation](https://tbnobody.github.io/OpenDTU-docs/hardware/esp32nodemcu_38pin/#schematic)
